src/bundle/Resources/config/routing_rest.yaml

```diff
@@ -36,6 +36,15 @@ ibexa.rest.location.tree.load_subtree:
     defaults:
         _controller: 'Ibexa\Bundle\AdminUi\Controller\Content\ContentTreeController::loadSubtreeAction'
 
+ibexa.rest.location.tree.load_node_extended_info:
+    # @todo change name to content tree
```

Comment:
Please resolve this `@todo`. It was left there because of BC for the other endpoint, let's not add more to technical debt. I'm not sure what the Author meant here, so this requires digging into git-blame and asking.

Reply:
It makes sense to have this TODO here. From what I understand, we would like to have in 5.0 a path such as: `/location/content-tree/...`. I can change this path now in 4.5 but I would make it inconsistent with previous paths, so I think we want it to be like it is now (`/location/tree/...`) in 4.5 and change it later, in 5.0 with other paths.